timestream.lua

```diff
@@ -4,17 +4,21 @@
 
 timestream
 ==========
-This only affects fortress mode.
+Controls the speed of the calendar and creatures. Fortress mode only. Experimental.
+
+The script is also capable of dynamically speeding up the game based on your current FPS to mitigate the effects of FPS death. See examples below to see how.
 
 Usage::
 
-    timestream <scalar> <fps> <simulate units y/n>
+    timestream <rate> <desired fps> <simulate units 1/0>
 
 Examples:
 
-:timestream     2:          Calendar runs at x2 normal speed
-:timestream    -1 100:      Calendar runs at dynamic speed to simulate 100 FPS
-:timestream    -1 100 1:    Calendar & units are simulated at 100 FPS
+    timestream     2:           Calendar runs at x2 normal speed, units run at normal speed
```

Comment:
Looks like you got rid of the leading colons here, which are needed to make it a "definition list" - it looks like this otherwise:
![image](https://user-images.githubusercontent.com/3719547/95284077-3d4cee80-082b-11eb-942d-ef33b8cdc799.png)

I will fix - just a heads-up for future work